run0/src/container/command.rs

```diff
@@ -0,0 +1,42 @@
+use oci_spec::runtime::Process;
+
+const UNSHARE_PROGRAM_ENTRYPOINT: &str = "/bin/sh";
+const DEFAULT_ARG0: &str = "sh";
+
+/// Implementation of the container arguments.
+#[derive(Default)]
+pub struct Command {
+    arg0: String,
+    args: Vec<String>,
+}
+
+impl Command {
+    /// Get arguments.
+    /// It converts the `Self` into a `Command` for `unshare` API compatibility.
+    pub fn get(&self) -> unshare::Command {
+        let mut command = unshare::Command::new(UNSHARE_PROGRAM_ENTRYPOINT);
+        command.arg0(&self.arg0).args(&self.args);
+
+        command
+    }
+}
+
+impl From<&Option<Process>> for Command {
+    fn from(process: &Option<Process>) -> Self {
```

Comment:
Here  you would create a default `Command`, now that you have a proper `Default`  implementation for that structure.

```rust
let mut command: Command = Default::default();
```

And then modify the default values if there are some args from the spec.